docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+Thank you for thinking about contributing to the WordPress sitemaps project! Contributions can either be made in the form of code or through input on tickets.
+We appreciate all contributions as they help to move the project forward.
+
+
+## Contributions to Issues
+
+Tickets will often need input from multiple points of view. We invite you to share your POV by adding a comment to the relevant ticket.
+
+If a ticket does not yet exist, you can create a new ticket and add it to the back log, but please do read: https://make.wordpress.org/core/2019/06/12/xml-sitemaps-feature-project-proposal/ and double check that what you are proposing is in line with what we are trying to achieve within this project's scope.
+
+
+### Creating an Issue
+
+- In Github, under 'Issues' click the green 'new issue' button.
+- Give the issue a short but meaningful title.
+- Give us much detail about the issue in the description box
+- Give the issue a label. Either:
+	- Type: Bug (Something isn't working)
+	- Type: Enhancement (Enhancement to an existing feature)
+	- Type: Feature (New feature)
+	- Type: Spike (Task to better understand a requirement)
+	- Type: Task (Specific task that may not be part of a given feature)
+- Assign the issue to the `Execution` project

```

Comment:
Yeah, this can be left out of the public eye. I would say external submissions all fall under a general "issue" and don't fall under any of the three categories here, nor do they need to worry about the project workflow, we take care of that in refinement.